web/vite.config.js

```diff
@@ -9,6 +9,19 @@ const config = {
 			'@api': path.resolve('./src/api')
 		}
 	},
+	server: {
+		// connect to a remote backend during web-only development
```

Comment:
Should we config this so it only applies when NODE_ENV=development?

Reply:
vite server is only ever used in development (`npm run dev`)